New Post - CSS `text-wrap: balance` (#5875)
GoogleChrome/developer.chrome.com: site/en/blog/css-text-wrap-balance/index.md

---
layout: layouts/blog-post.njk
title: "CSS text-wrap: balance"
description: >
  A classic typography technique of hand-authoring line breaks for balanced text blocks, comes to CSS.
subhead: >
  A classic typography technique of hand-authoring line breaks for balanced text blocks, comes to CSS.

> date: 2023-04-01

Author: ```suggestion
date: 2023-04-03
```

and then take a look at the examples in this post to learn how this one line of
CSS can massively improve your text layouts.

<figure>
  {% Video
    src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/qJKWQGssebOIDGVBtLpo.mp4",
    autoplay="true",
    loop="true",
    muted="true",
    controls="true"
  %}

  <figcaption>
    <a href="https://codepen.io/web-dot-dev/pen/KKxjpQm">
      Try a demo
    </a>
  </figcaption>
</figure>

Without `text-wrap: balance`; designers, content editors and publishers have
[few tools](https://developer.mozilla.org/docs/Web/CSS/CSS_Text/Wrapping_Text)
to change the way lines are balanced.. The best options available being to use
[`<wbr>`](https://developer.mozilla.org/docs/Web/HTML/Element/wbr) or
[`&shy;`](https://developer.mozilla.org/docs/Web/CSS/hyphens) to help
guide text layouts into smarter decisions about where to break lines and words.

As a developer, you don’t know the final size, font size, or even language of a
headline or paragraph. All the variables needed for an effective and aesthetic
treatment of text wrapping, are in the browser. This is why we see headline
wrapping as in the following image:

<figure>

>   {% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/tm4L941iouiU3Z2pj0Sw.png", alt="Headline is highlighted with DevTools, is spanning the full width of it's inline space, and has 2 hanging words on the 2nd line.", width="800", height="197" %}

Reviewer: ```suggestion
  {% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/tm4L941iouiU3Z2pj0Sw.png", alt="Headline is highlighted with DevTools, is spanning the full width of its inline space, and has two hanging words on the second line.", width="800", height="197" %}
```

---
layout: layouts/blog-post.njk
title: "CSS text-wrap: balance"
description: >
  A classic typography technique of hand-authoring line breaks for balanced text blocks, comes to CSS.
subhead: >
  A classic typography technique of hand-authoring line breaks for balanced text blocks, comes to CSS.
date: 2023-04-01
authors:
  - argyle
tags:
  - css
hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/7s9m7Sa2bx7OvH84PZZq.png
thumb: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/skq8r3b1NFR4B1gnm2an.png
alt: >
  A cairn of rocks balanced.
---

> [Chrome Canary](https://www.google.com/chrome/canary/) includes a new feature —
> `text-wrap: balance` from [CSS Text Level

Reviewer: ```suggestion
[Chrome Canary](https://www.google.com/chrome/canary/) includes a new feature—`text-wrap: balance` from [CSS Text Level
```
